src/Castle.Facilities.AspNetCore/Castle.Facilities.AspNetCore.csproj

```diff
@@ -0,0 +1,15 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+	<PropertyGroup>
+		<TargetFramework>netcoreapp2.0</TargetFramework>
+	</PropertyGroup>
+
+	<ItemGroup>
+		<PackageReference Include="Microsoft.AspNetCore.All" Version="2.0.5" />
```

Comment:
I can cut this right back. 

I just thought it would be handy if you created a console app, then installed Castle.Factilities.AspNetCore you would have everything ready go as a reference.

Reply:
Yes, I think we should cut it back.

The ASP.NET team just [announced today that they are actually introducing a new `Microsoft.AspNetCore.App` package](https://github.com/aspnet/Announcements/issues/287) and are changing their new project templates not to use ".All" in favour of this one because the "All" metapackage actually brings in heaps of stuff that isn't expected like Azure KeyVault, a Redis client and Sqlite. We really shouldn't force others to bring in packages we don't need for convenience.